generator/internal/discovery_file_test.cc

```diff
@@ -132,6 +132,8 @@ auto constexpr kGetRequestTypeJson = R"""({
 })""";
 
 TEST(DiscoveryFile, FormatFileWithImport) {
+  // TODO(sdhart): enable this when package PRs are finished.
```

Comment:
I think you intend this to be short-lived, but in case you win the lottery, please add an issue and reference it here.